### PR TITLE
Remove usage of TilesSchema byLocations endpoints

### PR DESCRIPTION
**Pull request by c-w**
This is a sister-change to pull request https://github.com/CatalystCode/project-fortis-interfaces/pull/10, see the the description of that change for further information.

### CODE REVIEW

File: src/components/Insights/HeatMap.js [hunk 1]
```diff
@@ -357,15 +348,14 @@ export const HeatMap = React.createClass({
...
     const zoom = this.map.getZoom();
     const bbox = this.getLeafletBbox();
     let self = this;
     this.weightedMeanValues = [];
 
     if(state.categoryValue.name){
-        SERVICES.getHeatmapTiles(siteKey, state.timespanType, zoom, state.categoryValue.name, state.datetimeSelection, 
-                                bbox, Array.from(state.termFilters), [state.selectedLocationCoordinates], Actions.DataSources(state.dataSource), 
+        SERVICES.getHeatmapTiles(siteKey, state.timespanType, zoom, state.categoryValue.name, state.datetimeSelection,
+                                bbox, Array.from(state.termFilters), undefined, Actions.DataSources(state.dataSource),
```
Comment (Smarker) [inline src/components/Insights/HeatMap.js]: Would it be possible to remove this undefined parameter? (see https://github.com/CatalystCode/project-fortis-interfaces/pull/10)

**Review by Smarker — APPROVED** [review]
LGTM - with comments

File: src/components/Insights/HeatMap.js (hunk 1 above)
Comment (c-w) [reply src/components/Insights/HeatMap.js]: Done in 18a7572.

**Review by Smarker — APPROVED** [review]
LGTM

**Review by erikschlegel — APPROVED** [review]
LGTM

**Pull request merged**